ignite/metrics/metrics_lambda.py

```diff
@@ -0,0 +1,44 @@
+from ignite.metrics.metric import Metric
+
+
+class MetricsLambda(Metric):
+    """
+    Apply a function to other metrics to obtain a new metric.
+    The result of the new metric is defined to be the result
+    of applying the function to the result of argument metrics.
+    For example, ``MetricsLambda(lambda x, y, z: x + y, metric1, metric2)``
```

Comment:
Where is the `z` coming from here? Should we maybe just remove this example in favor of the actual example below?